<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 255, 171);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>9</y>
      <width>681</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(147, 232, 255);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Agency FB&quot;;
background-color: rgb(169, 255, 184);</string>
      </property>
      <property name="text">
       <string>圆的半径</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Agency FB&quot;;
background-color: rgb(169, 255, 184);</string>
      </property>
      <property name="text">
       <string>圆的面积</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="radiusEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 255, 196);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="areaEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 255, 196);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calcButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>380</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>70</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Agency FB&quot;;
background-color: rgb(255, 171, 138);</string>
      </property>
      <property name="text">
       <string>圆的面积计算</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 99);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>长方形的长</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 99);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>长方形的宽</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>170</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 99);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>长方形的面积</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>55</y>
        <width>400</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 99);
font: 26pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>长方形的面积</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lengthEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(197, 255, 190);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="widthEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(197, 255, 190);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="areaEdit2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(197, 255, 190);
font: 16pt &quot;Agency FB&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calcButton2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>410</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>410</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="text">
       <string>敬请期待！</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="aButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 205, 255);</string>
    </property>
    <property name="text">
     <string>圆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 205, 255);</string>
    </property>
    <property name="text">
     <string>长方形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 205, 255);</string>
    </property>
    <property name="text">
     <string>未开发</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>计算的小工具</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
